about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="style.css">
+    <title>About</title>
+</head>
+<body id = 'about'>
+     <!-- first page for about page -->
+    <section>
+        <nav class="nav-top-desktop">
+            <ul>
+                <li><i class="fa-brands fa-facebook-f"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <div class="nav-top-mobile">
```

Comment:
- [ ] I think it would be great if you still have the hamburger menu in the about page since it is found in the about page so kindly add it there as well🙏🏻🤩🚀

- Your design
![image](https://user-images.githubusercontent.com/58553711/219305365-634ffa1c-d034-4dba-8f27-7c7deda1a27b.png)

- Given 
![image](https://user-images.githubusercontent.com/58553711/219305461-8a19f2d2-5024-498f-8027-aae98c6f23d7.png)